function-invokers/java-function-invoker/src/main/java/io/sk8s/invoker/java/server/JavaFunctionInvokerController.java

```diff
@@ -31,8 +31,8 @@
 	@Autowired
 	private FunctionProperties functions;
 
-	@PostMapping("/")
-	public Object invoke() {
+	@RequestMapping("/")
```

Comment:
In the context of sk8s, the only client of this is going to be the function sidecar, which we control. So I don't think we need/want to support GET.

Reply:
Fine by me. Spring Cloud Function supports GET. I'm still debating with @markfisher what that really means, but he has some ideas.

Reply:
For now, we should stick with `POST` here as that's the current (intentionally constrained) "contract" with the Sidecar when using the "http" dispatcher. We need to document said contract. We might add GET support when we consider the role of an event source "trigger" function/supplier.